### DISCUSSION TITLE
论文编译完成后pdf有两个相同标签

### DISCUSSION THREAD

**Discussion by wquqi** [sjtug/SJTUThesis#768] [提问]
如图，论文用compile.bat编译完成后，有两个“摘要”，第一个指向标题页，第二指向摘要，应该怎么解决呢？
![image](https://user-images.githubusercontent.com/38709654/154844584-a99303e8-247e-4fdf-9a5c-e40b4fbd8c59.png)

**Comment by skyzh** [comment]
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/4198311/154845942-8dd15735-9d36-478d-864f-c8d6afaecc0e.png">

macOS 上一切正常。您使用的是否是最新版？另外是本科模版还是研究生模版？

**Reply by wquqi** [reply]
我用的是老版本的研究生模板，因为新的版本里没有“攻读学位期间参与的项目”这一个附录，所以找师兄要了个旧的模板。请问在新的模板下怎么实现如下图那样，能够加入“项目”这个文档呢？
![image](https://user-images.githubusercontent.com/38709654/154902081-73569269-2857-4e17-a0a5-a942911a5754.png)

**Reply by wquqi** [reply]
新模板我编译了一遍，没有问题，windows平台用compile.bat编译

**Reply by skyzh** [reply]
老版本不维护了（

**Reply by skyzh** [reply]
直接照大摘要的方式加入应该就行了

**Reply by wquqi** [reply]
谢谢您的回答，我试下
